app/test_main.py

```diff
@@ -1,3 +1,92 @@
+from typing import Any, Type
+
+import pytest
+
 from app.main import get_human_age
 
-# write your code here
+
+class TestCatDogYears:
+    @pytest.mark.parametrize(
+        "cat_age,dog_age,human_age",
+        [
+            pytest.param(
+                0,
+                0,
+                [0, 0],
+                id="test should return zero ages when cat and dog age == 0"
+            ),
+            pytest.param(
+                14,
+                14,
+                [0, 0],
+                id="test should return zero ages when cat and dog age < 15"),
```

Comment:
This test case is redundant as it is testing the same condition as the test case at lines 13-17. According to the checklist, you should avoid testing the same case multiple times.